package.json

```diff
@@ -24,21 +24,18 @@
     "index.js"
   ],
   "devDependencies": {
-    "babel": "^6.1.18",
-    "babel-preset-es2015": "^6.1.18",
-    "babel-tape-runner": "^2.0.0",
+    "ava": "^0.11.0",
```

Comment:
Can you try adding [these tweaks](https://github.com/stylelint/stylelint-config-suitcss/commit/a0ac5c6e2e653f2842f9ba9879a71c7817525394) to your appveyor file please?

I'm not sure what you think, but perhaps testing on .12, 4 and 5 is a little excessive for a config. Probably just 5 is alright?